### PR TITLE
Realworld Example Application

### PR DESCRIPTION
**Pull request by agubler**
Implements the RealWorld Conduit app to the specs from https://github.com/gothinkster/realworld

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,18 +13,6 @@ For current progress on Dojo 2, contribution guidelines and other information, p
...
-
-```shell
-./scripts/gh-pages-publish.sh
-```
-
-The latest version(s) will now be available [on github pages](https://dojo.github.io/examples).
-
 ## Features
 
 There are currently six examples:
```
Comment (dylans) [inline README.md]: six -> seven

File: realworld/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# <img src="logo.png" width="48"> RealWorld Example App
+
+> ### Dojo 2 codebase containing real world examples (CRUD, auth, advanced patterns, etc) that adheres to the [RealWorld](https://github.com/gothinkster/realworld) spec and API.
+
+### [Demo](https://dojo.github.io/examples/realworld/)&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (dylans) [inline realworld/README.md]: Do we need &nbsp;&nbsp;&nbsp;&nbsp; at the end?

File: realworld/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+# How it works
+
+Dojo 2 RealWorld using @dojo/widget-core, @dojo/routing and @dojo/stores.
+
+For more information on Dojo 2 visit [dojo.io](https://dojo.io)
+
+# Getting started
+
+You can view a live demo over at https://dojo.github.io/examples/realworld/
```
Comment (dylans) [inline realworld/README.md]: You can view a live demo over at https://dojo.github.io/examples/realworld/ -> View the [Dojo 2 RealWorld live demo](https://dojo.github.io/examples/realworld/).

File: realworld/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+To get the frontend running locally:
+
+- Clone this repo
+- `npm install` to install all req'd dependencies
```
Comment (dylans) [inline realworld/README.md]: req'd -> required

File: README.md (hunk 1 above)
Comment (agubler) [reply README.md]: It's actually six, the cli-command example has been removed - I've removed that line.

**Comment by agubler** [comment]
@dylans addressed feedback

**Pull request merged**